README.md

```diff
@@ -58,7 +58,29 @@ being rate-limited first-hand!
 
 ### 2. Getting your credentials from GitHub
 
-You will need to register an OAuth application with GitHub by following these steps:
+There are two approaches to getting credentials from GitHub.
+First, you can get an access token. Second, you can register an OAuth app.
+The second approach is not recommended, and will be removed in a future release.
+
+#### Getting an access token
+
+This is the recommended way to authenticate yourself with GitHub.
+You can get an access token by following these steps:
+1. [Verify](https://help.github.com/articles/verifying-your-email-address) your email address with GitHub.
+1. Go to your account settings on GitHub and select "Developer Settings" from the left panel.
+1. On the left, select "Personal access tokens"
+1. Click the "Generate new token" button, and enter your password.
+1. Give the token a description, and check the "**repo**" scope box.
+1. Click "Generate token"
+1. You should be given a string which will be your access token.
+
+Remember that this token is effectively a password for your GitHub account.
+Do not share it online, as people can use it to access all of your data on GitHub.
+
+#### Setting up and OAuth application
+
+This approach to authenticating with GitHub is not recommended, and will be removed in a future release.
```

Comment:
You may wish to deprecate the approach proactively at this point and highlight with italics or bold.

Reply:
Thanks @willingc. I don't quite know what you mean by deprectate proactively? This is a traitlets config object, does that provide some deprecation mechanism?

Reply:
I meant though didn't state clearly a deprecation through words, instead of a formal code deprecation. Use words to convey ("Not supported", although it will work as follows in the current release, do not rely on this approach beyond that.)

Reply:
It is mentioned that it's not recommended above but it probably bears repeating here